src/app/pages/home/components/home-tabs.tsx

```diff
@@ -1,66 +1,51 @@
-import { Suspense } from 'react';
+import { Suspense, useCallback, useMemo } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
 
-import { Box, Flex, SlideFade, Stack } from '@stacks/ui';
+import { Box, Stack } from '@stacks/ui';
 import type { StackProps } from '@stacks/ui';
 
-import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
-import { useHomeTabs } from '@app/common/hooks/use-home-tabs';
+import { RouteUrls } from '@shared/route-urls';
+
 import { LoadingSpinner } from '@app/components/loading-spinner';
 import { Tabs } from '@app/components/tabs';
 
 interface HomeTabsProps extends StackProps {
-  balances: React.JSX.Element;
-  activity: React.JSX.Element;
+  children: React.JSX.Element;
 }
+// TODO: Abstract this to generic RouteTab once choose-fee-tab updated
+export function HomeTabs({ children }: HomeTabsProps) {
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
+
+  const tabs = useMemo(
+    () => [
+      { slug: RouteUrls.Home, label: 'Balances' },
+      { slug: RouteUrls.Activities, label: 'Activity' },
+    ],
+    []
+  );
 
-const ANALYTICS_PATH = ['/balances', '/activity'];
-
-export function HomeTabs(props: HomeTabsProps) {
-  const { balances, activity, ...rest } = props;
-  const analytics = useAnalytics();
-
-  const { activeTab, setActiveTab } = useHomeTabs();
+  const getActiveTab = useCallback(
+    () => tabs.findIndex(tab => tab.slug === pathname),
+    [tabs, pathname]
+  );
 
-  const setActiveTabTracked = (index: number) => {
-    void analytics.page('view', ANALYTICS_PATH[index]);
-    setActiveTab(index);
-  };
+  const setActiveTab = useCallback(
+    (index: number) => navigate(tabs[index]?.slug),
+    [navigate, tabs]
+  );
 
   return (
-    <Stack flexGrow={1} mt="loose" spacing="extra-loose" {...rest}>
+    <Stack flexGrow={1} mt="loose" spacing="extra-loose">
       <Tabs
-        tabs={[
-          { slug: 'balances', label: 'Balances' },
-          { slug: 'activity', label: 'Activity' },
-        ]}
-        activeTab={activeTab}
-        onTabClick={setActiveTabTracked}
+        tabs={tabs}
+        activeTab={getActiveTab()}
+        onTabClick={setActiveTab}
         width={['100%', '193px']}
       />
```

Comment:
Now we are using routes, I couldn't notice the `SlideFade` doing anything anymore so I removed it. Similarly the `Flex` container here seems superfluous although that could be incorrect.

Here's how it looks now:

https://github.com/hirosystems/wallet/assets/2938440/1fd84a48-d95b-4e85-bf80-bced49e3a5d1